### PR TITLE
Update annotator client

### LINKED ISSUE #151: Add ID and Contact to nlpsandboxclient
### Is your proposal related to a problem?

ID-annotator and contact-annotator specifications have not yet been incorporated into `nlpsandboxclient`.

### Describe the solution you'd like

There should be new functions `_annotate_contact()` and `_annotate_id()` in `nlpsandboxclient/client.py`, and new `if` statements under `annotate_note()` in the same file.

### PR DESCRIPTION
**Pull request by thomasyu888**
Fixes #151 
Depends on #155

### CODE REVIEW

File: nlpsandboxclient/client.py [hunk 1]
```diff
@@ -387,6 +504,13 @@ def annotate_note(host: str, note: Union[dict, Note],
             annotations = _annotate_person(api_client, text_annotator_req)
         elif tool_type == "nlpsandbox:physical-address-annotator":
             annotations = _annotate_address(api_client, text_annotator_req)
```
Comment (cascadianblue) [inline nlpsandboxclient/client.py:506]: were we going to update these method names to be more consistent with the `physical-address`, and `person-name` terminology?

**Review by boyleconnor — COMMENTED** [review]
LGTM, just left a comment on an issue I thought might be easier to address in one PR

File: nlpsandboxclient/client.py [hunk 2]
```diff
@@ -258,13 +261,13 @@ def _annotate_person(api_client, text_annotation_request: dict) -> dict:
         >>>        "text": "On 12/26/2020, Ms. Chloe Price met with Dr. Prescott."
         >>>    }
         >>> }
-        >>> host = "0.0.0.0:8080/api/v1"
+        >>> host = "0.0.0.0/api/v1"
         >>> configuration = annotator.Configuration(host=host)
         >>> with annotator.ApiClient(configuration) as api_client:
         >>>     annotations = _annotate_person(
```
Comment (thomasyu888) [inline nlpsandboxclient/client.py:267]:
```suggestion
        >>>     annotations = _annotate_person_name(
```

File: nlpsandboxclient/client.py [hunk 3]
```diff
@@ -294,13 +297,13 @@ def _annotate_address(api_client, text_annotation_request: dict) -> dict:
...
```
Comment (thomasyu888) [inline nlpsandboxclient/client.py:303]:
```suggestion
        >>>     annotations = _annotate_physical_address(
```

**Pull request merged**